hal/sparcv8leon3/cpu.h

```diff
@@ -54,6 +54,24 @@ static inline void hal_cpuDataStoreBarrier(void)
 }
 
 
+static inline void hal_cpuDataMemoryBarrier(void)
+{
+	/* not supported */
+}
+
+
+static inline void hal_cpuDataSyncBarrier(void)
+{
+	/* not supported */
+}
+
+
+static inline void hal_cpuInstrBarrier(void)
```

Comment:
This one look kinda like a `FLUSH` instruction

Reply:
`FLUSH` needs memory address as operand